### PR TITLE
chore: dont use db container if dbHost arg is overridden

### PR DESCRIPTION
**Pull request by drpebcak**

### CODE REVIEW

File: Acornfile [hunk 1]
```diff
@@ -10,11 +10,16 @@ args: {
 
 	// The tag that should be displayed in version information
 	tag: "v0.0.0-dev"
+
+	// The hostname of the database
+	dbHost: "test"
```
Comment (cloudnautique) [inline Acornfile:15]: To keep with the previous behavior, I'd make this whatever the local value is going to be.

File: Acornfile [hunk 2]
```diff
@@ -27,15 +32,15 @@ containers: {
...
+			ACORN_DB_HOST: args.dbHost
 			ACORN_DB_PORT: "3306"
 			ACORN_DB_NAME: args.dbName
 			ACORN_ROUTE53_ZONE_ID: args.route53ZoneId
 			AWS_ACCESS_KEY_ID: "secret://aws-creds/access-key?onchange=no-action"
 			AWS_SECRET_ACCESS_KEY: "secret://aws-creds/secret-key?onchange=no-action"
 		}
 	},
-
+    if args.dbHost == "db" {
```
Comment (cloudnautique) [inline Acornfile:43]: probably 6 of one half dozen of the other, but I'd do `!= local`. Leave it to you.

File: Acornfile (hunk 1 above)
Comment (drpebcak) [reply Acornfile]: Updated to be `db`, which should default to using the container.

File: Acornfile (hunk 2 above)
Comment (drpebcak) [reply Acornfile:43]: I'm going to leave this as it is for now, and refactor later as we adopt the profiles/services stuff.

**Review by cloudnautique — APPROVED** [review]

**Pull request merged**